### DISCUSSION TITLE
Buttons not Reskinning When Disabled

### DISCUSSION THREAD

**Discussion by Sythalin** [SFX-WoW/Masque#204] [Issues] [answered]
To reproduce:
1) Assign a skin to BT4
2) Place some button on a bar
3) Disable skinning to BT4 (on the option menu)
4) Move a button to another one

The "old" button should no longer have a backdrop/border.

![before](https://user-images.githubusercontent.com/4165258/65713055-ec25ee80-e05d-11e9-91e8-84b999292b61.png)
![after](https://user-images.githubusercontent.com/4165258/65713061-f0520c00-e05d-11e9-8cbe-1a0013d20eed.png)

**Comment by Sythalin** [comment]
To clarify, I disabled Masque and BT4 bars worked just fine and it doesn't seem to matter which skin is chosen.  The issue will clear on /reload, but if a button is moved it still does it.  Everything is fine BT4 is enabled/reenabled.

**Comment by StormFX** [comment, marked as answer]
This is actually an issue within LibActionButton-1.0. LAB doesn't register a callback with _Masque_, so isn't notified and doesn't update when a group is disabled or enabled.

**Comment by StormFX** [comment]
Closing this as an issue with [LibActionButton-1.0](https://www.wowace.com/projects/libactionbutton-1-0).
